### PR TITLE
Merge of Develop for Sprint 2 work

### PR DESCRIPTION
**Pull request by gervanpn**

### CODE REVIEW

File: big-game/src/app/competition-page/competition-page.component.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  }
+
+.gold-color {
+    color: rgb(224, 157, 11);
+    font-size: 22px;
+    padding-left: 5px;
+    line-height: 90%;
+}
+
+.mycard-light-green {
```
Comment (JuliaDunayeva) [inline big-game/src/app/competition-page/competition-page.component.css:109]: mycard again

File: big-game/src/app/competition-page/competitions/competitions.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+@Component({
+	selector: 'app-competitions',
+	templateUrl: './competitions.component.html',
+	styleUrls: [ './competitions.component.css' ]
+})
+export class CompetitionsComponent implements OnInit {
+	allBreeds: Breed[];
+	breedSelected: string;
+	compTypes = ['Trotting','Barrel Racing', 'Reining', 'Cutting', 'Trail Class', 'Western Pleasure'];
```
Comment (JuliaDunayeva) [inline big-game/src/app/competition-page/competitions/competitions.component.ts:14]: hardcoding

File: big-game/src/app/horse-list/horse-list.component.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    border-radius: 5px;
+}
+
+.td-line {
+    border-bottom: 2px solid rgb(192, 189, 113);
+    border-radius: 5px;
+}
+  
+.mycard-light-green {
+    background-color: #F0F0B2;
```
Comment (JuliaDunayeva) [inline big-game/src/app/horse-list/horse-list.component.css:80]: mycard again

File: big-game/src/app/horse-page/horse-page.component.css [hunk 4]
```diff
@@ -128,11 +129,211 @@ div>.card-body div>.card {
...
-#progressbar > div {
+#progressbar>div {
     background-color: orange;
     background: linear-gradient(90deg, rgba(242, 6, 6, 1) 0%, rgba(247, 244, 5, 1) 50%, rgba(0, 255, 29, 1) 100%);
     width: 70%;
     /* Adjust with JavaScript */
     height: 15px;
     border-radius: 10px;
 }
+
```
Comment (JuliaDunayeva) [inline big-game/src/app/horse-page/horse-page.component.css:140]: too much extra space

File: big-game/src/app/services/horse-data.service.ts [hunk 5]
```diff
@@ -1,18 +1,138 @@
...
+	getHorsesByUid() : Observable<HorseData[]>{
+		return this.db.collection('/horse_data', ref => ref.where('userId', '==', sessionStorage.getItem('uid')))
+		.snapshotChanges().pipe(
+			map(action => {
+			return action.map(res =>{
+				const horse=res.payload.doc.data() as HorseData;
+				const id=res.payload.doc.id;
+				//horse.id=id;
+				this.authService.sethorseId(id);
+				//console.log(horse.id);
```
Comment (JuliaDunayeva) [inline big-game/src/app/services/horse-data.service.ts:35]: clean up

File: big-game/src/app/competition-page/competitions/competitions.component.ts (hunk 2 above)
Comment (gervanpn) [reply big-game/src/app/competition-page/competitions/competitions.component.ts:14]: done by design - there is no db or intention for one at this point in the game for  types of competitions.  this is only to populate a drop-down with options

File: big-game/src/app/horse-list/horse-list.component.css (hunk 3 above)
Comment (gervanpn) [reply big-game/src/app/horse-list/horse-list.component.css:80]: resolved in next release

File: big-game/src/app/services/horse-data.service.ts (hunk 5 above)
Comment (gervanpn) [reply big-game/src/app/services/horse-data.service.ts:35]: see next version

**Pull request merged**